manifest.config.ts

```diff
@@ -50,6 +50,7 @@ export async function getManifest(opt: ManifestOptions): Promise<Manifest.WebExt
 			{
 				matches: ["*://*.twitch.tv/*"],
 				js: ["content.js"],
+				all_frames: true,
```

Comment:
Does this force a reinstall of the extension?

Reply:
No, at least as far as I've seen with testing it locally. It just enables 7tv to work in all twitch frames that aren't the top most frame (In this case it's the embedded iframe)

Is that what you're asking?

Reply:
![image](https://github.com/SevenTV/Extension/assets/49777269/9baf6d1f-c8b1-4571-918d-cab472d43317)

No, I am unsure if this constitutes as a change that will uninstall everyone's extensions so I'm scared to make it.

Reply:
Oh yeah no it won't, at least it shouldn't.

Reply:
It should just auto update whenever the extension is updated like normal

Reply:
I think, we wont fix this on this version and will wait for the rewrite due to be completed within the next few weeks.

Reply:
ok, should I just resubmit a pr then or will you guys plan on adding this feature in?